dispatch/BESRequestHandlerList.cc

```diff
@@ -107,12 +103,11 @@ BESRequestHandlerList::find_handler(const string &handler_name)
 {
     std::lock_guard<std::recursive_mutex> lock_me(d_cache_lock_mutex);
 
-    BESRequestHandlerList::Handler_citer i;
-    i = _handler_list.find(handler_name);
+    auto i = _handler_list.find(handler_name);
     if (i != _handler_list.end()) {
         return (*i).second;
```

Comment:
Based on other changes I think this could be:
```
return i->second;
```

Reply:
Yup. Changed